medicines/doc-index-updater/src/delete_manager/mod.rs

```diff
@@ -28,3 +51,39 @@ pub async fn get_message(delete_client: &mut Client) -> Result<String, AzureErro
 
     Ok(message)
 }
+
+pub async fn get_blob_name_from_message(
+    message: DeleteMessage,
```

Comment:
Can this signature be modified to take a content ID instead of the whole message? Will make it more reusable and easier to test.